code/game/objects/items/weapons/material/swiss.dm

```diff
@@ -115,16 +120,8 @@
 /obj/item/knife/folding/swiss/resolve_attackby(obj/target, mob/user)
 	if((istype(target, /obj/structure/window) || istype(target, /obj/structure/grille)) && active_tool == SWISSKNF_GBLADE)
 		force = force * 8
-		. = ..()
-		update_force()
-		return
-	if(istype(target, /obj/item))
```

Comment:
noting that this changes it so that it can work on things other than items that are normal-sized or smaller. which like, is fine, idrk why bay did it this way to begin with? but it's def a balance change